app/http/httpserver.yaml

```diff
@@ -33,7 +33,7 @@ components:
           x-field-uid: 3
         request_timeout:
           description: >-
-            The amount of seconds that the server will wait for input on an open connection before closing the session with a ‘408’ error.
+            The amount of seconds that the server will wait for input on an open connection before closing the session with a 408 error.
```

Comment:
The amount of seconds --> The number of seconds

Reply:
Updated in PR